### ISSUE TITLE
What API Group should we use for v1 of the operator?

### ISSUE THREAD

**Issue by carolynvs** [getporter/porter#2517]
When I originally wrote this, I used porter.sh/v1 for the API Group because that was our main domain for the project. Now that we have switched over to getporter.org, do we forsee any problems with using the different domain here for the API Group?

Just from a consistency perspective, it may make sense to cut over to getporter.org/v1. What do you think?

**Comment by VinozzZ** [comment]
I agree that even if it's just for the sake of consistency, it makes sense to change over to `getporter.org/v1`
